spec/index.bs

```diff
@@ -3942,6 +3958,45 @@ enum GPUCullMode {
 };
 </script>
 
+#### Alpha to Coverage #### {#alpha-to-coverage}
+
+In alpha-to-coverage mode, which is enabled by {{GPURasterizationStateDescriptor/alphaToCoverageEnabled}},
+an additional <dfn dfn>alpha-to-coverage mask</dfn>
+of MSAA samples is generated based on the |alpha| component of the
+fragment shader output value of the {{GPURenderPipelineDescriptor/colorStates}}[0].
```

Comment:
Old content, but reading this again, is this how this works? Vulkan spec kind of makes it sound like it's done per-attachment.
https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/chap29.html#fragops-covg

Reply:
See:
> All alpha values in this section refer only to the alpha component of the fragment shader output that has a Location and **Index decoration of zero** (see the Fragment Output Interface section). If that shader output has an integer or unsigned integer type, then these operations are skipped.

Reply:
Oops, got it